### PR TITLE
the star block is created

### PR DESCRIPTION
**Pull request by vanyachyzh**
- [DEMO LINK](https://vanyachyzh.github.io/layout_stars/)
- [TEST REPORT LINK](https://vanyachyzh.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  width: 100px;
```
Comment (OlEzhA0) [inline src/style.css:7]: you might not add width style

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  width: 16px;
+  background: url("./images/star.svg") no-repeat center;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5),
+.stars--6 .stars__star:nth-child(-n+6) {
```
Comment (OlEzhA0) [inline src/style.css:23]: are you sure you have `stars--6`?)

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey! please check my comments

**Review by witflash — APPROVED** [review]
Well done 👍

**Pull request closed without merging**